polyfill/lib/zoneddatetime.mjs

```diff
@@ -393,8 +393,13 @@ export class ZonedDateTime {
       'getOffsetNanosecondsFor',
       'getPossibleInstantsFor'
     ]);
-    const offsetNs = ES.GetOffsetNanosecondsFor(timeZoneRec, GetSlot(this, INSTANT));
-    const dt = ES.GetPlainDateTimeFor(timeZoneRec, GetSlot(this, INSTANT), GetSlot(this, CALENDAR), offsetNs);
+
+    const calendar = GetSlot(this, CALENDAR);
+    const instant = GetSlot(this, INSTANT);
+    const thisNs = GetSlot(instant, EPOCHNANOSECONDS);
+
+    const offsetNs = ES.GetOffsetNanosecondsFor(timeZoneRec, instant);
+    const dt = ES.GetPlainDateTimeFor(timeZoneRec, instant, calendar, offsetNs);
```

Comment:
Mind splitting this out to an editorial PR? As far as I can tell, DRY-ing the `GetSlot` calls isn't observable. Delegates reviewing for TC39 are going to want to see the minimal set of normative changes.